packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/exportInclusiveLanguageConfigSpec.js

```diff
@@ -79,9 +79,10 @@ describe( "Export of the inclusive language configuration", () => {
 		resultLines.unshift( header.join( ";" ) );
 
 		// Set doExport to true to write the results to a temporary file.
-		const doExport = false;
+		const doExport = true;
 		if ( doExport ) {
 			writeToTempFile( "inclusive-language-database.csv", resultLines.join( "\n" ) );
+			console.log("result=== \n"+resultLines.join( "\n" ));
```

Comment:
Please fix the formatting of this line, and temporarily disable the eslint rule that adds a warning for `console.log` statements (see https://stackoverflow.com/a/34215841).